### ISSUE TITLE
Distribution tag in RPMs

### ISSUE THREAD

**Issue by andral** [seveas/python-hpilo#99]
First of all a big thank you for this awesome library, it makes our lives a lot easier!

Would you mind adding a distribution tag to the rpms?
Without it the rpms are named the same way for different distributions (el6, el7 etc) but are actually different. Repository management systems like pulp will throw an error because they have different checksums.

```
 $ wget http://download.opensuse.org/repositories/home:/seveas:/python/RHEL_6/noarch/python-hpilo-3.4-1.1.noarch.rpm -O python-hpilo-3.4-1.1.el6.noarch.rpm
 $ wget http://download.opensuse.org/repositories/home:/seveas:/python/RHEL_7/noarch/python-hpilo-3.4-1.1.noarch.rpm -O python-hpilo-3.4-1.1.el7.noarch.rpm
 $ md5sum python-hpilo*
f8d2c562e1c94c20d1e99a55d6d76994  python-hpilo-3.4-1.1.el6.noarch.rpm
76eacda2c8eada1102900cbdc4754ab8  python-hpilo-3.4-1.1.el7.noarch.rpm
```

In your spec file you can add the tag as follows.

```
Name:           python-hpilo
Version:        3.4
Release:        1%{?dist}
```

which would result in python-hpilo-3.4-1.el7.noarch for example.

Thanks

**Comment by seveas** [comment]
Yeah, that makes sense. Will do. Also note that the OBS repository is going away soon as OBS isn't working very well. It will be replaced with a fedora copr repository.

**Comment by andral** [comment]
Thanks for the quick response and the heads up!

**Comment by seveas** [comment]
Fixed in 3.6
